ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestValues.scala

```diff
@@ -41,6 +41,7 @@ private[backend] object StorageBackendTestValues {
     ParameterStorageBackend.IdentityParams(someLedgerId, someParticipantId)
   val someParty: Ref.Party = Ref.Party.assertFromString("party")
   val someApplicationId: Ref.ApplicationId = Ref.ApplicationId.assertFromString("application_id")
+  val someSubmissionId: String = "submission_id"
```

Comment:
Can you make this a `Ref.SubmissionId`? All other `some*` values are properly typed as well.

Reply:
87da1dd